tapir/coop/templates/coop/email/membershipresignation_confirmation_body.html

```diff
@@ -2,73 +2,34 @@
 {% load i18n %}
 {% load utils %}
 {% block body %}
-    (English below)
-    <p>Liebe*r {% get_display_name_short share_owner %},</p>
-    <p>
-        schade, dass Du uns verlassen möchtest. Hiermit bestätigen wir Dir die Kündigung deiner Mitgliedschaft bei SuperCoop Berlin eG.
-    </p>
+    {% get_display_name_short share_owner as display_name %}
+    {% get_display_name_full resigned_member.transferring_shares_to as transferred_member %}
```

Comment:
We need to move this line (`{% get_display_name_full resigned_member.transferring_shares_to as transferred_member %}`) inside `{% elif resigned_member.resignation_type == resignation_type_transfer %}` otherwise we send `None` to `get_display_name_full` and that crashes